api4/file.go

```diff
@@ -18,6 +18,17 @@ const (
 	FILE_TEAM_ID = "noteam"
 )
 
+var IMAGE_CONTENT_TYPES = [...]string{
```

Comment:
misleading name: these aren't just image content types. "media" or "safe" might be more accurate